daq/acl_state_collector.py

```diff
@@ -0,0 +1,70 @@
+"""Process ACL states"""
+
+from __future__ import absolute_import
+
+import logger
+
+LOGGER = logger.get_logger('acl')
```

Comment:
aclstate?  I think "acl" might be too short to be "memorable"...

Reply:
updated